packages/search/lib/utils.js

```diff
@@ -21,8 +28,15 @@ const getIndexAlias =
  * @param  {String} name An index name without prefix
  * @return {String}      An index name with a date timestamp
  */
-const getIndexDated =
-  (name) => [ES_INDEX_PREFIX, name, sanitizedDate()].filter(Boolean).join('-')
+const getDatetimeIndex =
+  (name) => [ES_INDEX_PREFIX, name, getDateTime()].filter(Boolean).join('-')
+
+/**
+ * @param  {String} name An index name without prefix
+ * @return {String}      An index name with a date
+ */
+const getDatedIndex =
```

Comment:
I would like `getDateIndex` better

Reply:
Yes, adopting. Was under the impression, originally, that it might lead to confusion.